client/src/visualiser-src/binary-search-tree-visualiser/initialiser.ts

```diff
@@ -9,35 +9,23 @@ export interface BSTVisualiser extends Visualiser {
   rotateRight: (val: number, updateSlider: (val: number) => void) => void;
 }
 
-/**
- * TODO: documentation
```

Comment:
We could probably add a link to the confluence page as a temporary substitute.

Reply:
Sounds good :+1: 

Here's the link: https://compclub.atlassian.net/wiki/spaces/S/pages/2150892071/Documentation#Visualiser-Docs%3A

Reply:
:+1: